cmake-integration.el

```diff
@@ -43,11 +43,29 @@
 ;; TODO: Change this to a custom variable
 (defvar cmake-integration-build-dir "build" "The build folder to use when no presets are used. If this is nil, then using presets is required")
 
-(defvar cmake-integration-current-target-name nil)
+(defvar cmake-integration-current-target nil)
 (defvar cmake-integration-current-target-run-arguments "")
 
 (defvar cmake-integration-last-configure-preset nil)
 
+(defconst cmake-integration--multi-config-separator "/"
+  "Character used to separate target name from config name.
+
+In case of of multi-config generators, target names have the
+special form <target-name><sep><config-name> (e.g. all/Debug with '/' as
+configured separator).
+
+Note: The selected separator shall be a character that it is not
+a valid component of a CMake target name (see
+https://cmake.org/cmake/help/latest/policy/CMP0037.html).")
+
+
+(defun cmake-integration--mktarget (target-name &optional config-name)
+  "Return target constructed from from TARGET-NAME and CONFIG-NAME."
+  (if config-name
+      (concat target-name cmake-integration--multi-config-separator config-name)
+    (target-name)))
```

Comment:
Here in the "else" part you need to replace `(target-name)` with `target-name`, since it's a variable and not a function.

Reply:
fixed